lib/kubernetes-deploy/template_sets.rb

```diff
@@ -59,7 +59,7 @@ def validate
 
       private
 
-      def templates(filename:)
+      def templates(filename:, raw:)
```

Comment:
Could you explain the semantics of `raw` for me, please? It's not entirely clear to me what the intent of it is.

Reply:
Currently this function returns a ruby object that results from parsing the yaml. `raw` means I wan't the string that comes directly from the erb rendering. 

This is useful for 2 reasons over just calling `Yaml.dump` on the ruby object:
 1. it preserves the whitespace & comments.  
 2. You can put duplicate keys into a hash in yaml. Because ruby doesn't support this (nor does any language I know of), it silently throws away the first once which means when you go to lint it, it will pass even though its not valid.